angular-sample-spa/README.md

```diff
@@ -10,22 +10,95 @@ The IBM Cloud AppID SDK can be used with React to create a secure single-page ap
 
 ## To run locally
 
-* Clone the repository
+1 Clone the repository.
 ```
-git clone 
+git clone https://github.com/ibm-cloud-security/appid-sample-code-snippets.git
 ```
-* Navigate to application workspace folder.
+2 Navigate to application workspace folder.
 ```
-cd angular-sample-spa
+cd angular-sample-app
 ```
-* To get the dpendencies for the app installed run
+3 Install dependencies for the application.
 ```
 npm install
 ```
-
-* To start the development server run 
+4 Start the development server. Navigate to http://localhost:4200 to access your application.
 ```
 ng serve
 ```
 
 The app will automatically reload if you change any of the source files.
+
+## Detailed instructions on Creating you application
```

Comment:
This section is a little confusing. Are you expecting users to do these actions in this repo or in their own workspace from a new directory? Also typo in `Detailed instructions on Creating you application`

Reply:
Maybe rename to `Detailed instructions on adding App ID to sample Angular App` or something similar that captures that users are creating a new app and adding app id to it

Reply:
This shouldn't be here